main_induction.py

```diff
@@ -123,13 +134,13 @@ def sorted_models(model_scores):
 
 
     model_scores = {}
-    kfold = KFold(n_splits=NUM_K_FOLD_SPLITS)
+    kfold = StratifiedKFold(n_splits=NUM_K_FOLD_SPLITS)
```

Comment:
this isn't super important but apparently it helps balance out the label classes per-fold